### PR TITLE
Updated p2-repo with apim store/publisher feature categories

### PR DESCRIPTION
**Pull request by lalaji**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -47,10 +47,31 @@
     </pluginRepositories>
 
     <properties>
+        <axis2.transport.version.1.1.0v15>1.1.0-wso2v15</axis2.transport.version.1.1.0v15>
+        <am.version.2.0.0>2.0.0-SNAPSHOT</am.version.2.0.0>
```
Comment (Niranjan-K) [inline pom.xml]: shouldn't this be the carbon apim repo version since we do not refer to the product?

Comment (lalaji) [inline pom.xml]: This is the carbon apim version.Change the label name.

File: pom.xml [hunk 2]
```diff
@@ -47,10 +47,31 @@
     </pluginRepositories>
 
     <properties>
+        <axis2.transport.version.1.1.0v15>1.1.0-wso2v15</axis2.transport.version.1.1.0v15>
+        <carbon.apimgt.version.2.0.0>2.0.0-SNAPSHOT</carbon.apimgt.version.2.0.0>
```
Comment (Niranjan-K) [inline pom.xml]: PR is put on hold till carbon apimgt 2.0.0 is released and the version is updated here.

File: pom.xml [hunk 3]
```diff
@@ -72,9 +93,273 @@
...
+                                <featureArtifactDef>org.jaggeryjs.modules:caramel.feature:${caramel.feature.version.1.4.0}</featureArtifactDef>
+                                <featureArtifactDef>org.jaggeryjs.modules:carbon.feature:${carbon.feature.version.1.4.0}</featureArtifactDef>
+                                <featureArtifactDef>org.jaggeryjs.modules:handlebars.feature:${handlebars.feature.version.1.4.0}</featureArtifactDef>
+                                <featureArtifactDef>org.jaggeryjs.modules:i18n.feature:${i18n.feature.version.1.4.0}</featureArtifactDef>
+                                <featureArtifactDef>org.jaggeryjs.modules:markdown.feature:${markdown.feature.version.1.4.0}</featureArtifactDef>
+                                <featureArtifactDef>org.jaggeryjs.modules:process.feature:${process.feature.version.1.4.0}</featureArtifactDef>
+                                <featureArtifactDef>org.jaggeryjs.modules:sso.feature:${sso.feature.version.1.4.0}</featureArtifactDef>
+                                <featureArtifactDef>org.jaggeryjs.modules:uuid.feature:${uuid.feature.version.1.4.0}</featureArtifactDef>
```
Comment (Niranjan-K) [inline pom.xml]: This is not in alphabetical order. Jaggery features need to move up.

File: pom.xml [hunk 4]
```diff
@@ -47,10 +47,31 @@
     </pluginRepositories>
 
     <properties>
+        <axis2.transport.version.1.1.0v15>1.1.0-wso2v15</axis2.transport.version.1.1.0v15>
```
Comment (Niranjan-K) [inline pom.xml:50]: This property file is not used anywhere and can be removed.

File: pom.xml [hunk 5]
```diff
@@ -47,10 +65,39 @@
...
         <carbon.p2.plugin.version>1.5.4</carbon.p2.plugin.version>
-
+        <carbon.registry.version.4.4.8>4.4.8</carbon.registry.version.4.4.8>
+        <carbon.rule.version.4.4.0>4.4.0</carbon.rule.version.4.4.0>
+        <carbon.social.version.2.3.4>2.3.4</carbon.social.version.2.3.4>
+        <cipher.tool.version.1.0.0v3>1.0.0-wso2v3</cipher.tool.version.1.0.0v3>
+        <es.feature.version.2.3.4>2.3.4</es.feature.version.2.3.4>
+        <handlebars.feature.version.1.4.0>1.4.0</handlebars.feature.version.1.4.0>
+        <i18n.feature.version.1.4.0>1.4.0</i18n.feature.version.1.4.0>
+		<jaggery-test.feature.version.1.2.2>1.2.2</jaggery-test.feature.version.1.2.2>
```
Comment (Niranjan-K) [inline pom.xml:74]: I think there is an alignment issue here.

File: pom.xml [hunk 6]
```diff
@@ -72,9 +119,390 @@
...
+                                <!--feature categories-->
+
+                                <featureArtifactDef>
+                                    org.wso2.carbon.apimgt:org.wso2.apimgt.store.category.feature:${carbon.apimgt.version.2.0.0}
+                                </featureArtifactDef>
+                                <featureArtifactDef>
+                                    org.wso2.carbon.apimgt:org.wso2.apimgt.publisher.category.feature:${carbon.apimgt.version.2.0.0}
+                                </featureArtifactDef>
+
+                                <featureArtifactDef>
```
Comment (Niranjan-K) [inline pom.xml:441]: These are greg feature and should be above the "Feature Categories"

**Pull request merged**